hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -2763,20 +2765,20 @@ public boolean processReport(final DatanodeID nodeID,
       if (namesystem.isInStartupSafeMode()
           && !StorageType.PROVIDED.equals(storageInfo.getStorageType())
           && storageInfo.getBlockReportCount() > 0) {
-        blockLog.info("BLOCK* processReport 0x{}: "
+        blockLog.info("BLOCK* processReport 0x{} with fullBrLeaseId 0x{}: "
```

Comment:
@tomscut Thanks for the PR.

`fullBrLeaseID` is a variable name. I prefer to use "lease ID" in the log message.
```suggestion
        blockLog.info("BLOCK* processReport 0x{} with lease ID 0x{}: "
```